labs/lab-1-html-and-css/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link href='http://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" href="desktop-stylesheet.css">
+  <link rel="stylesheet" href="stylesheet.css">
+  <!--
```

Comment:
Ditto here. Your credit is in the *commit*, and doesn't belong in the file. Have you been looking at some old-school code somewhere? This is a very 1990's thing to do :)

Reply:
I just noticed these comments, ha. They made us do it in our Java class so I thought that was some kind of standard. Good to know! Thanks for the feedback!